pylzy/lzy/py_env/py_env_provider.py

```diff
@@ -87,6 +87,9 @@ def search(obj: Any) -> None:
             if name in STDLIB_LIST or name in sys.builtin_module_names:
                 return
 
+            if name in exclude_packages:
```

Comment:
I'd recommend to cast exclude_packages to set at some place for explicit O(1) in